app/models/ordered_trip.rb

```diff
@@ -2,4 +2,5 @@ class OrderedTrip < ActiveRecord::Base
   belongs_to :order
   belongs_to :pursuit
 
+  validates_numericality_of :travellers, greater_than: 0, allow_blank: true
```

Comment:
Prefer the new style validations `validates :column, numericality: value` over `validates_numericality_of`.